From: Jarkko Sakkinen <jarkko@kernel.org>
Subject: MAINTAINERS: TPM DEVICE DRIVER: update the W-tag
Date: Tue, 25 Nov 2025 18:03:53 +0200

I migrated test suite to git.kernel.org so that all my kernel stuff is
consolidated to one single place:

https://git.kernel.org/pub/scm/linux/kernel/git/jarkko/linux-tpmdd-test.git/about/

Link: https://lkml.kernel.org/r/20251125160353.2300402-1-jarkko@kernel.org
Signed-off-by: Jarkko Sakkinen <jarkko@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/MAINTAINERS~maintainers-tpm-device-driver-update-the-w-tag
+++ a/MAINTAINERS
@@ -26048,7 +26048,7 @@ M:	Jarkko Sakkinen <jarkko@kernel.org>
 R:	Jason Gunthorpe <jgg@ziepe.ca>
 L:	linux-integrity@vger.kernel.org
 S:	Maintained
-W:	https://codeberg.org/jarkko/linux-tpmdd-test
+W:	https://git.kernel.org/pub/scm/linux/kernel/git/jarkko/linux-tpmdd-test.git/about/
 Q:	https://patchwork.kernel.org/project/linux-integrity/list/
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/jarkko/linux-tpmdd.git
 F:	Documentation/devicetree/bindings/tpm/
_
